mlir/include/mlir/Dialect/Linalg/Transforms/Transforms.h

```diff
@@ -1089,7 +1089,7 @@ collapseOpIterationDims(LinalgType op,
 
 struct LowerPackResult {
   tensor::PadOp padOp;
-  tensor::ExpandShapeOp expandShapeOp;
+  Operation *expandShapeOp;
```

Comment:
Not actually sure what would be appropriate here.  Alternatively, we could have two separate fields for the `ExpandShapeOp` and `ReshapeOp`, but I haven't looked into the implications of this yet.  I welcome suggestions.